pkg/util/containers/container_linux.go

```diff
@@ -75,6 +80,10 @@ func (c *Container) FillCgroupMetrics() error {
 	if err != nil {
 		return fmt.Errorf("start time: %s", err)
 	}
+	c.ThreadCount, err = c.cgroup.ThreadCount()
```

Comment:
Nothing against this change, I'm just wondering if this method `FillCgroupMetrics()` should return `[]error` of an aggregated list of errors.
Because currently if the first check return an error, the others values are not set. WDYT?

same for `FillCgroupLimits()`

Reply:
All "expected" errors (missing file, limit not set... see `ThreadLimit` for example) are handled by the individual methods. This is pretty much the "abort the mission" case.